fixtures/chauvet-professional/colorado-1-solo.json

```diff
@@ -2,11 +2,11 @@
   "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
   "name": "COLORado 1 Solo",
   "shortName": "Solo 1",
-  "categories": ["Dimmer", "Color Changer", "Strobe"],
+  "categories": ["Color Changer"],
   "meta": {
-    "authors": ["Da Graca Neves Eric"],
+    "authors": ["Da Graca Neves Eric", "Ken Harris"],
     "createDate": "2021-01-17",
-    "lastModifyDate": "2021-01-17"
+    "lastModifyDate": "2022-03-27"
```

Comment:
Please update both dates to today (or whenever you read this :wink:), because the create date should be the merge date.